bndtools.core/src/bndtools/central/Central.java

```diff
@@ -212,6 +215,20 @@ public synchronized static Workspace getWorkspace() throws Exception {
             newWorkspace.addBasicPlugin(new WorkspaceListener(newWorkspace));
             newWorkspace.addBasicPlugin(instance.repoListenerTracker);
             newWorkspace.addBasicPlugin(getWorkspaceR5Repository());
+
+            // need to remove any progress plugins that already exist so we can favor JobProgress
+            Set<Object> progressPlugins = new HashSet<>();
+
+            for (Object plugin : newWorkspace.getPlugins()) {
+                if (plugin instanceof ProgressPlugin) {
+                    progressPlugins.add(plugin);
+                }
+            }
+
+            for (Object plugin : progressPlugins) {
+                newWorkspace.removeBasicPlugin(plugin);
+            }
+
```

Comment:
Also I've removed the previous commit that removed the progress plugins, instead now its adds JobProgress since https://github.com/bndtools/bnd/pull/1525 adds support for multiple progress plugins